src/streaming/protection/drm/KeySystemW3CClearKey.js

```diff
@@ -0,0 +1,114 @@
+/**
```

Comment:
Could you please add unit tests for this class? That helps us a lot to detect possible regression in future releases.